sdk/src/main/scala/com/horizen/api/http/SidechainBlockApiRoute.scala

```diff
@@ -20,7 +19,7 @@ import scala.collection.JavaConverters._
 import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
-case class SidechainBlockApiRoute(override val settings: RESTApiSettings, sidechainNodeViewHolderRef: ActorRef, sidechainBlockActorRef: ActorRef, forgerRef: ActorRef, forgingControlRef: ActorRef)
+case class SidechainBlockApiRoute(override val settings: RESTApiSettings, sidechainNodeViewHolderRef: ActorRef, sidechainBlockActorRef: ActorRef, forgingControlRef: ActorRef)
```

Comment:
Verify the names of `forgingControlRef: ActorRef` consistency in current class constructor and object apply/props methods with the names chosen in `BlockActor`, `SidechainApiRouteTest` and `SidechainApp`.
Different names for the same thing used: `forgingControlRef`/`sidechainForgingControlRef`/`forgerRef`/`sidechainBlockForgerActorRef`